Jenkinsfile_CNP

```diff
@@ -13,8 +13,8 @@ def branchesToSync = ['demo', 'ithc', 'perftest', 'develop']
 GradleBuilder builder = new GradleBuilder(this, product)
 
 // Variables to switch pipeline logic and wiring per type of build
-def definitionStoreDevelopPr = "PR-575"  // This doesn't change frequently, but when it does, only change this value.
-def dataStoreApiDevelopPr    = "PR-1260" // This doesn't change frequently, but when it does, only change this value.
+def definitionStoreDevelopPr = "PR-1392"  // This doesn't change frequently, but when it does, only change this value.
```

Comment:
Revert after def-store and data-store merged

Reply:
Reverted